PP_APP/server/frontend/templates/about.html

```diff
@@ -44,107 +44,109 @@ <h2 class="align-items-center text-center">&nbsp;</h2>
+                can find the code on our <a target="_blank" href=https://github.com/kennylee9672/ECS193A>Github</a>.
+                <br />
+            </p>
+        </div>
+
         <!-- Project Two Row -->
-        <div class="row justify-content-center no-gutters">
-            <div class="col-lg-6">
-                <img class="img-fluid" src="{% static 'img/demo-image-02.jpg' %}" alt="">
-            </div>
-            <div class="col-lg-6 order-lg-first">
+        <div class="row justify-content-center no-gutters py-3">
+            <div class="col-lg-6 order-lg-first px-3">
                 <div class="bg-black text-center h-100 project">
                     <div class="d-flex h-100">
                         <div class="project-text w-100 my-auto text-center text-lg-right">
-                            <h4 class="text-white">Eric Law</h4>
-                            <p class="mb-0 text-white">An example of a project with its respective description. These
-                                sections work well responsively as well, try this theme on a small screen!</p>
+                            <h4 class="text-white">Yoosuf Shafi</h4>
+                            <p class="mb-0 text-white">
+                               <a class="about-link" target="_blank" href="https://github.com/kennylee9672">Github</a> |
+                               <a class="about-link" target="_blank" href="https://www.linkedin.com/feed/">LinkedIn</a></p>
                             <hr class="d-none d-lg-block mb-0 mr-0">
                         </div>
                     </div>
                 </div>
             </div>
-        </div>
-        <h2 class="align-items-center text-center">&nbsp;</h2>
-        <!-- Project One Row -->
-        <div class="row justify-content-center no-gutters mb-5 mb-lg-0">
-            <div class="col-lg-6">
-                <img class="img-fluid" src="{% static 'img/demo-image-02.jpg' %}" alt="">
-            </div>
-            <div class="col-lg-6">
-                <div class="bg-black text-center h-100 project">
+
+            <div class="col-lg-6 px-3">
+                <div class="bg-black-rev text-center h-100 project">
                     <div class="d-flex h-100">
                         <div class="project-text w-100 my-auto text-center text-lg-left">
-                            <h4 class="text-white">Yoosuf Shafi</h4>
-                            <p class="mb-0 text-white">An example of where you can put an image of a project, or
-                                anything else, along with a description.</p>
+                            <h4 class="text-white">Kenny Li</h4>
+                            <p class="mb-0 text-white">
+                              <a class="about-link" target="_blank" href="https://github.com/kennylee9672">Github</a> |
+                              <a class="about-link" target="_blank" href="https://www.linkedin.com/feed/">LinkedIn</a>
+                            </p>
```

Comment:
Please fix the Github and Linkedin links for both Kenny and I.

Reply:
My apologies, fixed